Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:2
+
+ADD . /usr/src/app
```

Comment:
`COPY` is preferred over add, ref the [best practices guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy).  Also this, copies the README.md into the location as well, which is necessary, so just be explicit.